scripts/build.py

```diff
@@ -285,10 +361,19 @@ def get_font_filepaths(dependency_bundle, dependency_dir):
         list(str). List of paths to font files that need to be copied.
     """
     if 'fontsPath' not in dependency_bundle:
+        # Skipping dependency bundle in manifest.json that does not have
+        # fontsPath property.
         return []
     fonts_path = dependency_bundle['fontsPath']
-    return [os.path.join(dependency_dir, fonts_path, extension)
-            for extension in FONT_EXTENSIONS]
+    # Obtain directory path to /font inside dependency folder.
+    font_dir = os.path.join(dependency_dir, fonts_path)
+    # E.g. third_party/static/bootstrap-3.3.4/fonts/.
+    font_filepaths = []
+    # Walk the directory and add all fonts file to list.
+    for root, _, files in os.walk(font_dir):
```

Comment:
Here, should we be talking about filenames rather than files? Or filepaths?

Again please make sure to distinguish correctly between files, filepaths and filenames throughout:
- files are the file objects
- filepaths are the full path to the file object, like /a/b/c/abc.mp3
- filenames are the last part of the filepath, like abc.mp3

Reply:
Basically, I was just trying to use the same naming convention for `os.walk` throughout build.py. If we change it here, we will be changing it for the rest of the files wherever uses `os.walk`

Reply:
Changing them elsewhere is fine with me, too. As long as things are communicated clearly and consistently.